kolibri/plugins/html5_app_renderer/assets/src/vue/index.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+
+  <div v-el:container class="container" allowfullscreen>
+    <icon-button
+      class="btn"
+      :text="inFullscreen ? $tr('exitFullscreen') : $tr('enterFullscreen')"
+      @click="togglefullscreen">
+    </icon-button>
+    <iframe v-el:sandbox class="sandbox" :src="indexpath" sandbox="allow-scripts"></iframe>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+
+    components: {
+      'icon-button': require('kolibri.coreVue.components.iconButton'),
+    },
+
+    props: ['defaultFile'],
+
+    data: () => ({
+      inFullscreen: false,
+    }),
+
+    computed: {
+      indexpath() {
+        return this.defaultFile.storage_url + "index.html";
+      }
+    },
+
+    methods: {
+      togglefullscreen() {
```

Comment:
If all content had the option, then "Enter Fullscreen" could be down alongside "Download Content", rather than taking up a row for itself.